tests/src/testcommon.hh

```diff
@@ -278,8 +259,181 @@ template <typename NonLinearOperator, typename FiniteElement>
   return t;
 }
 
+template <typename NonLinearOperator, typename FiniteElement>
+[[nodiscard]] auto checkLinearStressOf3DSimplex(NonLinearOperator& nonLinearOperator, FiniteElement& fe,
```

Comment:
I really like generic things but if this really is a 3Dsimplex, we should write tetraheadron.